spec/requests/api/v0/manifests_search_spec.rb

```diff
@@ -1,35 +1,31 @@
 require 'rails_helper'
-require_relative '../../db/chores/populator'
 require "queryparams"
 
 describe "search", elasticsearch: true do
+  before(:each) do
+    Manifest.rebuild_index
```

Comment:
I'd probably prefer this to be inlined. If it essential to a test passing correctly, having it inline tells me what the dependencies of the test are.